src/runner/utils.ts

```diff
@@ -10,8 +10,12 @@ import { simple } from '../utils/walkers'
 
 // Context Utils
 
-export function isFullJSChapter(chapter: number): boolean {
-  return chapter == -1
+export const isOtherLanguage = (chapter: number) => {
```

Comment:
Perhaps it will be clearer if we have a `isSource`/`isSourceLanguage` instead? My concern is that `OtherLanguage` might be a little vague